src/Rides.java

```diff
@@ -0,0 +1,52 @@
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Rides {
+
+
+    ArrayList<Double> fare_total = new ArrayList<>();
+    ArrayList<Double> id_ride_final1 = new ArrayList<>();
+    private double fare_estimate;
+    public Rides (){
+
+        this.fare_estimate = fare_estimate;
+
+    }
+
+    public String prepare_result (ArrayList<Double> fare, ArrayList<Double> id_ride_final, int i) {
+
+         int  n = 0, m=0;
+         double  cont =1.0;
+        String result = "0";
+
+         System.out.println("id_ride | fare_estimate");
+
+        while (cont <= id_ride_final.get(i-1)) {
+
+          fare_estimate = 0;
```

Comment:
fare_estimate should be initialize in 1.3 and not in 0. Then you can remove this line 


          fare_estimate = (fare_estimate + 1.30);